### DISCUSSION TITLE
Lots of great mods not in the official repo

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk-mods#500] [General]
There are lots of the great mods whose authors do not submit them into the repo nor they specify a license. Below is a list of some.

* [Enabling Windows Classic theme per process](https://winclassic.boards.net/thread/1967/mod-enables-classic-theme). This method does not need permission to inject system processes, but fails to untheme some dialogs and Conhost. Useful when you want to untheme only one process or untheme all except one.
* [Classic Explorer Toolbars](https://winclassic.boards.net/thread/1976/hide-navigation-classic-explorer-toolbars). Brings back the Windows XP-style toolbars and throbbler.
* [Disable folder thumbnails](https://winclassic.net/post/27830)
* [Winver customizer](https://winclassic.boards.net/thread/2113/winver-customizer)
* [Remove custom UWP titlebars](https://winclassic.net/thread/2041/remove-windows-10s-uwp-titlebars)
* [Normal Mica on taskbar](https://winclassic.net/thread/1931/normal-mica-on-taskbar-win11)
* [Square Caption Buttons](https://winclassic.net/thread/2131/square-caption-buttons) - for visual styles imitating Classic theme (but not real Classic theme). Makes the window caption buttons square.
* [Chrome: Bypass Save As dialog](https://github.com/valinet/wh-mods/blob/master/mods/valinet-chrome-save-to-downloads-directly.wh.cpp)
* [Custom Keyboard Mapping](https://github.com/valinet/wh-mods/blob/master/mods/valinet-custom-keyboard.wh.cpp)
* [Custom actions when chassis power button is pressed](https://github.com/valinet/wh-mods/blob/master/mods/valinet-power-button-action.wh.cpp)
* [Change display brightness by scrolling over the battery icon](https://github.com/valinet/wh-mods/blob/master/mods/valinet-systray-battery-brightness.wh.cpp)
* [Change system volume by scrolling over the sound icon](https://github.com/valinet/wh-mods/blob/master/mods/valinet-systray-sound-volume.wh.cpp)
* [Make server OS report as client OS](https://github.com/valinet/wh-mods/blob/master/mods/valinet-unserver.wh.cpp)
* [Fix crash of audio service due to Waves Audio](https://github.com/valinet/wh-mods/blob/master/mods/valinet-waves-audio-fix.wh.cpp)
* [Adjusting blurs present throughout the system](https://github.com/im-tem/acrylic-mica-tweaker)
* [Enable dark titlebar for win32 app when system darkmode is on](https://github.com/fifv/windhawk_mods/blob/master/dark-window-titlebar.cpp)
* [Control Panel Color Fix version 1.0](https://winclassic.net/thread/3288/control-panel-color-fix-version) - this version is compatible with the Classic theme, unlike the later version 1.0.1 from the repository.

**Comment by m417z** [comment]
I think it's best to have all mods in the official repository, it's more convenient for users to install, update, and see insights such as the amount of users. Unfortunately, I can't force authors to submit their mods. If you're active on the relevant forums, it would be great if you could ask the authors to submit the mods, help them with it if necessary, or at least ask what's the reason for them to avoid submitting.

**Reply by slrgt** [reply]
Is there documentation on how to submit mods?

**Reply by Anixx** [reply]
Go to https://github.com/ramensoftware/windhawk-mods/tree/main/mods and click "Add file". Then enter the mod file name (like "mode-name.wh.cpp") and the mod code. Then just click the green submit buttons (twice, first to your branch, then to the main repo).

**Comment by Anixx** [comment]
First thing to do is to ensure new mods use license tags. Then anyone can submit them into the main repo.

**Reply by m417z** [reply]
It can help, but ideally, it's best for the authors to submit their mod.
